pom.xml

```diff
@@ -57,6 +57,7 @@
               <xmlOutput>true</xmlOutput>
               <findbugsXmlWithMessages>true</findbugsXmlWithMessages>
               <failOnError>${findbugs.failOnError}</failOnError>
+              <omitVisitors>UnreadFields</omitVisitors>
```

Comment:
I failed to have FindBugs suppress a false positive for a field used in the Groovy view only, so used this workaround.